perfectscale/manifest.json

```diff
@@ -0,0 +1,56 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "1f6c7afa-b008-4fa6-a82d-8259c9c423e3",
+  "app_id": "perfectscale",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Receive PerfectScale optimization alerts directly in Datadog to expedite remediations.",
+    "title": "PerfectScale",
+    "media": [
```

Comment:
Can you include images of the dashboard as well?

Reply:
Done